README.md

```diff
@@ -87,26 +133,25 @@ The possible values are:
 	
 The successfully verified message of the example would then look like this:
 ```
-# UUID                          # timestamp     # some value
 4f6b64a7a5c9483786c00d32bc8e03c080d1a6763192d01500003fc93178a59b00
+|<           UUID             >||< timestamp  >||<      misc    >|
+|          (16 byte)           ||   (8 byte)   ||     (9 byte)   |
```

Comment:
the verifier appends a byte-long code to the original message. if you say the original message is 16+8+X bytes long, the verifier responds with a message thats 16+8+X+2 bytes long (not 9 byte)